its/src/test/java/com/sonar/it/scanner/msbuild/utils/TestUtils.java

```diff
@@ -242,13 +242,18 @@ public static void runMSBuildWithBuildWrapper(Orchestrator orch, Path projectDir
     assertThat(r).isZero();
   }
 
-  public static void runMSBuild(Orchestrator orch, Path projectDir, String... arguments) {
-    runMSBuild(orch, projectDir, Collections.emptyList(), arguments);
+  public static BuildResult runMSBuild(Orchestrator orch, Path projectDir, String... arguments) {
+    return runMSBuild(orch, projectDir, Collections.emptyList(), 60 * 1000, arguments);
```

Comment:
NITPICK 60 * 1000 is used in multiple places, please introduce a constant for it.